### DISCUSSION TITLE
build error: GitVersion.MsBuild.targets

### DISCUSSION THREAD

**Discussion by Delari** [coinfoundry/miningcore#1255] [General]
**Describe the bug**
C:\Users\Delari\.nuget\packages\gitversion.msbuild\5.8.2\tools\GitVersion.MsBuild.targets(9,9): error MSB3073: exit from command "dotnet --roll-forward Major "C:\Users\Delari\.nuget\packages\gitversion.msbuild\5.8.2\tools\netcoreapp3.1/gitversion.dll" "C:\Users\Delari\source\repos\miningcore\src\Miningcore" -output file -outputfile obj\gitversion.json" with code 1. [C:\Users\Delari\source\repos\miningcore\src\Miningcore\Miningcore.csproj]

**To Reproduce**
1. download (git clone)
2. run build bat-script or build on VS 2022

**LOG**
PS C:\Users\Delari\source\repos\miningcore> .\build-windows.bat
Microsoft (R) Build Engine version 17.1.1+a02f73656 for .NET

  Determining projects to restore...
  All projects are updated for restoration.

  Cannot find the .git directory
C:\Users\Delari\.nuget\packages\gitversion.msbuild\5.8.2\tools\GitVersion.MsBuild.targets(9,9): error MSB3073: exit from command "dotnet --roll-forward Major "C:\Users\Delari\.nuget\packages\gitversion.msbuild\5.8.2\tools\netcoreapp3.1/gitversion.dll" "C:\Users\Delari\source\repos\miningcore\src\Miningcore" -output file -outputfile obj\gitversion.json" with code 1. [C:\Users\Delari\source\repos\miningcore\src\Miningcore\Miningcore.csproj]
PS C:\Users\Delari\source\repos\miningcore> dotnet --info
Package SDK for.NET (reflecting any global.json):
 Version:   6.0.202
 Commit:    f8a55617d2

Runtime:
 OS Name:     Windows
 OS Version:  10.0.19044
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\6.0.202\

Host (useful for support):
  Version: 6.0.4
  Commit:  be98e88c76

.NET SDKs installed:
  6.0.202 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 3.1.10 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.4 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.10 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.4 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 6.0.4 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download

**Dotnet-gitversion**

PS C:\Users\Delari\source\repos\miningcore> dotnet-gitversion -version
5.10.1+Branch.support-5.x.Sha.b17d33e3aef55c573f03d95d975557b5fcaf6591
PS C:\Users\Delari\source\repos\miningcore>

**Comment by eshcrow** [comment]
1) First you have to install [GitHub Desktop](https://desktop.github.com/)
2) Reboot ypu PC
3) While holding down the Shift key, right-click on the source directory (not inside the directory, but on the folder icon) and select "open PowerShell here", and then follow the developer's instructions.
After performing these actions, there will be no error.
